modules/actions/circularize.js

```diff
@@ -1,15 +1,15 @@
-import { median as d3_median } from 'd3-array';
+import {median as d3_median} from 'd3-array';
 
 import {
     polygonArea as d3_polygonArea,
-    polygonHull as d3_polygonHull,
-    polygonCentroid as d3_polygonCentroid
+    polygonCentroid as d3_polygonCentroid,
+    polygonHull as d3_polygonHull
 } from 'd3-polygon';
 
-import { geoVecInterp, geoVecLength } from '../geo';
-import { osmNode } from '../osm/node';
-import { utilArrayUniq } from '../util';
-import { geoVecLengthSquare } from '../geo/vector';
+import {geoVecInterp, geoVecLength} from '../geo';
+import {osmNode} from '../osm/node';
+import {utilArrayUniq} from '../util';
+import {geoVecLengthSquare} from '../geo/vector';
```

Comment:
As you can see in most of iD's source files, the existing spacing is preferred, so could you change these back?

Reply:
Ah – sorry, my editor gets a bit aggressive at times.